deployment/GP02HYPM_Deploy.csv

```diff
@@ -1,99 +1,99 @@
+MV1404,,TN323,,GP02HYPM-WFP03-05-VEL3DL000,2,1,2014-06-22T02:14:00,2015-06-02T00:00:00,CGMGP-02HYPM-00002,CGVEH-GWFPGX-10402,CGINS-VEL3DL-13551,50.07833,-144.80533,,4038,4219,
+#MV1404,,TN323,,GP02HYPM-MPM01-02-ZPLSGA009,2,1,2014-06-22T02:14:00,2015-06-02T00:00:00,CGMGP-02HYPM-00002,,,50.07833,-144.80533,,,4219,Not Deployed
+#MV1404,,TN323,,GP02HYPM-MPM01-02-ZPLSGA010,2,1,2014-06-22T02:14:00,2015-06-02T00:00:00,CGMGP-02HYPM-00002,,,50.07833,-144.80533,,,4219,Not Deployed
+TN323,,RB1605,,GP02HYPM-RIM01-00-SIOENG000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,,CGCON-SIOCON-11690,50.07983,-144.803,,4080,4219,
+TN323,,RB1605,,GP02HYPM-RIM01-02-CTDMOG039,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,,CGINS-CTDMOG-10257,50.07983,-144.803,,163,4219,
+TN323,,RB1605,,GP02HYPM-WFP02-00-WFPENG000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-99701,CGCON-GWFPGX-99701,50.07983,-144.803,,2102,4219,
+TN323,,RB1605,,GP02HYPM-WFP02-01-FLORDL000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-99701,CGINS-FLORDL-02934,50.07983,-144.803,,2102,4219,
+TN323,,RB1605,,GP02HYPM-WFP02-03-DOSTAL000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-99701,CGINS-DOSTAL-01105,50.07983,-144.803,,2102,4219,
+TN323,,RB1605,,GP02HYPM-WFP02-04-CTDPFL000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-99701,CGINS-CTDPFL-00115,50.07983,-144.803,,2102,4219,
+TN323,,RB1605,,GP02HYPM-WFP02-05-VEL3DL000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-99701,CGINS-VEL3DL-01026,50.07983,-144.803,,2102,4219,
+TN323,,RB1605,,GP02HYPM-WFP03-00-WFPENG000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-10403,CGCON-GWFPGX-10403,50.07983,-144.803,,4070,4219,
+TN323,,RB1605,,GP02HYPM-WFP03-01-FLORDL000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-10403,CGINS-FLORDL-03284,50.07983,-144.803,,4070,4219,
+TN323,,RB1605,,GP02HYPM-WFP03-03-DOSTAL000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-10403,CGINS-DOSTAL-01478,50.07983,-144.803,,4070,4219,
+TN323,,RB1605,,GP02HYPM-WFP03-04-CTDPFL000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-10403,CGINS-CTDPFL-00132,50.07983,-144.803,,4070,4219,
+TN323,,RB1605,,GP02HYPM-WFP03-05-VEL3DL000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-10403,CGINS-VEL3DL-01115,50.07983,-144.803,,4070,4219,
+#TN323,,RB1605,,GP02HYPM-MPM01-02-ZPLSGA009,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,,,50.07983,-144.803,,,4219,Not Deployed
+#TN323,,RB1605,,GP02HYPM-MPM01-02-ZPLSGA010,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,,,50.07983,-144.803,,,4219,Not Deployed

```

Comment:
Deployment 3: Correction looks good.
Single recovery time is correct – mid-water release failed, bottom release valid for all components.